src/extension/views/doc-source/js/ui.js

```diff
@@ -247,8 +248,9 @@ const load = async () => {
     sendMessage({ fct: 'setMain', params: { html: htmlEditionToSource() } });
   }, 500));
 
-  sampleRUM('sidekick:viewdocsource', {
-    source: tab.url,
+  sampleRUM('click', {
+    source: 'sidekick',
+    target: 'viewdocsource',
```

Comment:
Should we change this to past tense to match the others, e.g. `doc-source-copied`?